mock/redfish/v1/UpdateService/FirmwareInventory/NIC-drvs/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "@odata.id": "/redfish/v1/UpdateService/FirmwareInventory/NIC-drvs",
+  "Description": "The information on NIC drivers.",
+  "Id": "NIC-drvs",
+  "Name": "NIC-drvs",
+  "SoftwareId": "NIC-45-HJK",
+  "Version": "5.32.5"
+}
```

Comment:
Do you think it's realistic that a multi-system server would not have a `RelitedItem` property to point at the system where this NIC is located? Or is this to simulate such cases?